### ISSUE TITLE
Using android:singleLine and android:MaxLines:2 at the same time does not work like what i intended

### ISSUE THREAD

**Issue by JuneBuug** [grantland/android-autofittextview#75]
I am using this autoFitTextView 

```
<me.grantland.widget.AutofitTextView
                    android:id="@+id/counsel_info"
                    android:layout_width="match_parent"
                    android:layout_height="wrap_content"
                    android:layout_marginLeft="5dp"
                    android:text="test"
                    android:textColor="@color/gray_color"
                    autofit:minTextSize="10sp"
                    android:singleLine="true"
                    android:maxLines="2"
                    />
```

There are two cases i wanted to solve: 

1) I want to the text to be 2 lines if it is too long to be shown in a one line, expecting it to be shown in full text. (even if the text is in its minTextSize)

2) Also, I need this text to be shown in a one line when it is long enough to be resized, but still can be shown in one line( even if the text is in its minTextSize or bigger) 

However, when i applied the code as shown above, the first case i want to solve was shown in a one line. 

When i got singleLine code out, second case started to be shown in a two lines. 

How can i solve this problem?

In short, how can i set autofitTextview to try to resize text first, and when it does not work use another line?

**Comment by grantland** [comment]
Unfortunately this is not a use case we support. Additionally, singleLine and maxLines works just as documented in AOSP `TextView` javadocs.
